tests/appsec/test_traces.py

```diff
@@ -329,7 +330,7 @@ class Test_DistributedTraceInfo:
     """Distributed traces info (Services, URL, trace id)"""
 
     def test_main(self):
-        assert False, "Test not implemented"
+        pytest.skip("Test not implemented")
```

Comment:
same

Reply:
Can you either remove, or use official decorators ? 

Same for later `pytest.skip`